4_ASP.NET_CORE/task4/NLAYER_server/FilmsCatalog/FilmsCatalog/WebApi/HttpBase/HttpContextBaseController.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.Linq;
+using System.Security.Claims;
+
+namespace FilmsCtalog.WebApi.HttpBase
+{
+    public static class HttpContextBaseController
```

Comment:
That is extension class,  originally I mean base class for your controllers inherited from ControllerBase